project/Common.scala

```diff
@@ -56,7 +56,8 @@ object Common extends AutoPlugin {
       "-deprecation",
       "-Xlint",
       "-Ywarn-dead-code",
-      "-target:jvm-1.8"),
+      "-target:jvm-1.8",
+      "-Wconf:cat=unused-nowarn:s"),
```

Comment:
This is needed because we use the `@nowarn` annotation to suppress a warning only for Scala 2.13 and without this scalac option it would complain that no warnings were suppressed for Scala 2.12.